styles.css

```diff
@@ -557,3 +565,60 @@ h4 {
   margin-top: 35px;
   border-bottom: 1px solid #e0d8d8;
 }
+
+/* form-section */
+
+.form {
+  background-image: url(./images/Contact-me.jpg);
+  height: 779px;
+}
+
+.form h2 {
+  color: #fff;
+  display: flex;
+  justify-content: center;
+  padding-top: 102px;
+  font-size: 40px;
+}
+
+.form p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+  padding: 0 20px 20px 20px;
+  margin-top: 12px;
+}
+
+form {
+  padding-left: 24px;
+}
+
+.form label {
+  display: inline-block;
+  width: 90px;
+}
+
+input,
```

Comment:
- [ ] It will be better if you add some padding to your inputs 
![real](https://user-images.githubusercontent.com/80895497/155171567-335c6ead-f03d-46f7-a00a-529b13402a28.PNG)